docs/reference.rst

```diff
@@ -167,6 +167,12 @@ hierarchical separator.
   Marks the slot as existing but not updatable. May be used for sanity checking
   or informative purpose. A ``readonly`` slot cannot be a target slot.
 
+``ignore-checksum``
```

Comment:
Maybe:

> If set to ``true`` this will bypass the default hash comparison for this slot and force RAUC to unconditionally update it. The default value is ``false`` which means that updating this slot will be skipped if new image's hash matches hash of installed one.

Reply:
Yes much better :)